clients/javascript/game/cell.js

```diff
@@ -41,10 +42,12 @@ module.exports = class Cell {
 
   actions() {
     const actions = this.actions_;
-    actions.target = this.target;
+    if(this.target !== this.target_) {
```

Comment:
I'm not 100% sure (I honestly don't remember). If this `if` wasn't there, every turn you be considered as a move. But now, `move` is only considered if we actually do a `move()`.